services/scan/src/end_to_end.test.ts

```diff
@@ -42,27 +42,18 @@ afterEach(async () => {
 });
 
 test('going through the whole process works', async () => {
-  // Do this first so interpreter workers get initialized with the right value.
   await request(app)
-    .patch('/central-scanner/config/skipElectionHashCheck')
-    .send({ skipElectionHashCheck: true })
```

Comment:
Not sure why we needed to do this first, the tests pass if we do it after setting the election.